Demo/Common/Configuration/CardComponentSettingsView.swift

```diff
@@ -20,8 +20,13 @@ internal struct CardComponentSettingsView: View {
                     Toggle(isOn: $viewModel.showsHolderNameField) {
                         Text("Holder Name")
                     }
+                    
                     Toggle(isOn: $viewModel.showsStorePaymentMethodField) {
-                        Text("Stored Payment Method")
+                        VStack(alignment: .leading) {
+                            Text("Stored Payment Method")
+                            Text("(v70 and above)")
```

Comment:
Maybe tiny improvement to give it more context
```suggestion
                            Text("Requires API version 70 or higher")
```

Reply:
f4620ff8087f903eef8bdf0735b0f91493a2009b